server/db/models/product.js

```diff
@@ -26,6 +26,11 @@ const Product = db.define('product', {
   }
 })
 
+Product.prototype.getPrice = function() {
+  let thisPrice = this.price
+  return thisPrice * 0.01
+}
```

Comment:
This looks great. Do we need a setter, as well? Or will we do the conversion client-side?

Reply:
conversion is client-side because We want to keep this stored as an int.